gn3/computations/correlations.py

```diff
@@ -168,8 +166,14 @@ def compute_all_sample_correlation(this_trait,
     for target_trait in target_dataset:
         trait_name = target_trait.get("trait_id")
         target_trait_data = target_trait["trait_sample_data"]
-        this_vals, target_vals = list(zip(*list(filter_shared_sample_keys(
-            this_trait_samples, target_trait_data))))
+
+        try:
+            this_vals, target_vals = list(zip(*list(filter_shared_sample_keys(
+                this_trait_samples, target_trait_data))))
+
+        except ValueError:
+            # case where no matching strain names
+            this_vals, target_vals = []
```

Comment:
**issue**: This generates an error.

You can't do:

```
a, b = []
```

This will generate this error:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: not enough values to unpack (expected 2, got 0)
```

Reply:
made the recommended changes